test/runtests.jl

```diff
@@ -43,6 +43,18 @@ T = Float32
         @test isapprox( signalsum, T(2), atol = 5e-3 )
         nt = NamedTuple(sim)
         @test sim == Simulation(nt)
+        id = 2
+        deplV = get_depletion_voltage(sim, id, verbose = false)
+        @info deplV
+        @test isapprox(deplV, T(1904), atol = 2.0) 
+        # Check wether detector is undepleted, 10V below the previously calculated depletion voltage
+        sim.detector = SolidStateDetector(sim.detector, contact_id = id, contact_potential = deplV - 40)
+        calculate_electric_potential!(sim, depletion_handling = true)
+        @test !is_depleted(sim.point_types)
+        # Check wether detector is depleted, 10V above the previously calculated depletion voltage
+        sim.detector = SolidStateDetector(sim.detector, contact_id = id, contact_potential = deplV + 40)
+        calculate_electric_potential!(sim, depletion_handling = true)
+        @test is_depleted(sim.point_types)
```

Comment:
We might wanna use finer `refinement_limits` for this test to get closer to the "real" depletion voltage and maybe really only check 10V above and below (rather than the 40V used here).

Reply:
I think this won`t solve the issue, because the underlying problem is, that the point types are fully depleted, when the global min max search is done. Still the local min max search returns something bigger than that. This delta will always determine how much below we will have to check, and this delta can actually be quite high (around 50V sometimes)